.gitignore

```diff
@@ -3,4 +3,5 @@
 build/*
 dist/*
 data/*
-__pycache__
\ No newline at end of file
+__pycache__
+plepy.egg-info/*
```

Comment:
add newline at end of file or you get these weird notices in github. Generally a good practice. Check your IDE settings to see if it can ensure there is always a trailing new line.

you can just use *.egg-info here and it will be general for any app you write

Reply:
I fixed these and updated my IDE to ensure there's one trailing newline at the end of files.